src/sentry/lang/javascript/processor.py

```diff
@@ -545,7 +545,10 @@ def preprocess_step(self, processing_task):
 
     def handles_frame(self, frame, stacktrace_info):
         platform = frame.get("platform") or self.data.get("platform")
-        return settings.SENTRY_SCRAPE_JAVASCRIPT_CONTEXT and platform in ("javascript", "node")
+        has_context_line = bool(frame.get("context_line"))
+        # if the frame already has a context_line, we consider it already symbolicated
+        # and therefore skip the processor handling it
+        return settings.SENTRY_SCRAPE_JAVASCRIPT_CONTEXT and platform in ("javascript", "node") and is_valid_frame(frame) and not has_context_line
```

Comment:
If we move the new condition earlier, we don't have to evaluate the rest of the expression (including a function call) when `has_context_line` is `True`.

```suggestion
        return (not has_context_line) and settings.SENTRY_SCRAPE_JAVASCRIPT_CONTEXT and platform in ("javascript", "node") and is_valid_frame(frame)
```